dvc/command/repro.py

```diff
@@ -35,6 +35,10 @@ def run(self):
                     metrics = self.repo.metrics.show()
                     logger.info(_show_metrics(metrics))
 
+                logger.info(
+                    "Use 'dvc push' to send your updates to remote storage."
+                )
```

Comment:
Two more things:
1. please use ` instead of ' - to keep consistency with other help messages
2. I think we should move this message few lines up and log it under `if len(stages)==0` in `else` cause, so that we will not get this message when there is nothing actually reproduced.

What do you think @jvfe?

Reply:
Sorry, I don't really understand what you mean by using "instead of" in this context, could you explain further?

Also, sounds good putting an else statement below if `len(stages)==0`, like this, right?

```py
if len(stages) == 0:
    logger.info(CmdDataStatus.UP_TO_DATE_MSG)
else:
    logger.info(
        "Use 'dvc push' to send your updates to remote storage."
    )
```

Reply:
@jvfe I mean backticks (  **`**  ) instead of ticks (  **'**  ) - sorry its hard to highlit them, due to markdown formatting.

Reply:
Ok, thanks for explaining! I've done what you suggested but had to split the message in two lines or the pre-commit hook for flake8 would complain.